projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.html

```diff
@@ -16,7 +16,7 @@
 </div>
 <button
   *ngIf="hasStock"
-  class="btn btn-primary btn-block cx-btn"
+  class="btn btn-primary btn-block btn-relative"
```

Comment:
This is would be a breaking change since this file is over a year old, but it's also not needed to fix the bug.